exonum/src/proto/schema/exonum/blockchain.proto

```diff
@@ -29,7 +35,7 @@ message Block {
   exonum.crypto.Hash tx_hash = 5;
   exonum.crypto.Hash state_hash = 6;
   exonum.crypto.Hash error_hash = 7;
-  OrderedMap additional_headers = 8;
+  AdditionalHeaders additional_headers = 8;
```

Comment:
Changing the type without changing the field number leads to breaking backward compatibility. Is it fine?
See: https://stackoverflow.com/questions/27362742/changing-data-type-for-a-field-in-protocol-buffer

Reply:
This field was introduced after the `0.13.0-rc.2` version (see [blockchain.proto](https://github.com/exonum/exonum/blob/v0.13.0-rc.2/exonum/src/proto/schema/exonum/blockchain.proto#L23) on release tag), thus changing the type between releases is perfectly fine.

Reply:
@bullet-tooth I think it's fine; we've never released a version with `OrderedMap`; it was added after the 0.13 RC.